src/app/auth/auth-registration/auth-registration.component.ts

```diff
@@ -0,0 +1,259 @@
+import { Component, OnInit } from "@angular/core";
+import { Router, ActivatedRoute, ParamMap } from "@angular/router";
+import { UtilsService } from "@services/utils.service";
+import { Md5 } from "ts-md5/dist/md5";
+import {
+  Validators,
+  FormControl,
+  FormGroup,
+  FormBuilder
+} from "@angular/forms";
+
+import { AuthService } from "../auth.service";
+import { BrowserStorageService } from "@services/storage.service";
+
+@Component({
+  selector: "app-auth-registration",
+  templateUrl: "./auth-registration.component.html",
+  styleUrls: ["./auth-registration.component.scss"]
+})
+export class AuthRegistrationComponent implements OnInit {
+  password: string;
+  confirmPassword: string;
+  isAgreed: boolean = false;
+  registerationForm: FormGroup;
+  hide_password: boolean = false;
```

Comment:
If I change it to true, it still ask me to fill the required field
![image](https://user-images.githubusercontent.com/14796879/49191049-1a4c8a80-f3c9-11e8-9ad5-5e84c9213453.png)

Reply:
i changed validation for this.